source/configuration_manual/mail_location/Maildir/index.rst

```diff
@@ -40,6 +40,12 @@ you'll need to enable fs layout:
 
 .. _maildir_settings_control_files:
 
+``FULLDIRNAME`` Default
+-----------------------
+
+For Maildir, the default for the
+:ref:`FULLDIRNAME key <mail_location_settings-keys>` is <empty>.
```

Comment:
This seems a bit weird way of listing defaults. How about rather something like:

```
Default mail_location keys
--------------------------

For Maildir, the default <mail_location_settings-keys> are:

 * FULLDIRNAME: <empty>
 * Maybe some others listed as well?..
```

Reply:
Changed this on all of the pages